policykit/integrations/slack/views.py

```diff
@@ -241,10 +241,10 @@ def action(request):
                 new_api_action.channel = event['channel_id']
                 new_api_action.timestamp = event['item']['message']['ts']
 
-        if new_api_action.initiator.has_perm('slackintegration.add_' + new_api_action.action_codename):
+        if new_api_action.initiator.has_perm('slack.add_' + new_api_action.action_codename):
             if new_api_action and not policy_kit_action:
                 #if they have execute permission, skip all policies
```

Comment:
@amyxzhang @rickygv99 this code block seems like it is doing the same thing that happens in PlatformAction `save` here: https://github.com/amyxzhang/policykit/blob/c6bcb88240d1bd79e8b01743222ab2fd96883dac/policykit/policyengine/models.py#L779-L816

Is there a reason this permissions check is duplicated in Slack integrations (but not in the other integrations)?

Reply:
I can't think of any good reason it might be duplicated in just the Slack integration. Since the Slack integration was built before being encapsulated in the more general integration framework, I'm tempted to say it's old purposeless code.